src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.cs

```diff
@@ -153,7 +156,28 @@ public static unsafe bool IsSortable(string text)
 
             fixed (char* pChar = text)
             {
-                return IsSortable(pChar, text.Length);
+                return IsSortableCore(pChar, text.Length);
+            }
+        }
+
+        private static unsafe bool IsSortableCore(char* pChar, int length)
```

Comment:
>IsSortableCore(char* pChar, int length) [](start = 35, length = 39)

nit: make it the style 

IsSortableCore(char* pChar, int length) => GlobalizationMode.UseIcu ? IcuIsSortable(pChar, length) :
 NlsIsSortable(pChar, length);

Reply:
maybe you need to apply this to other similar methods.

---
In reply to: [405682662](https://github.com/dotnet/runtime/pull/34645#discussion_r405682662) [](ancestors = 405682662)